roles/ansible/tower/manage-organizations/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- block:
+    - name: "Set default values"
+      set_fact:
+        existing_organizations_output: []
+        existing_credentials_output: []
+        processed_organizations: []
```

Comment:
this fact doesn't seem to be used for anything, so can probably be cleaned-out(?)

Reply:
same for the `existing_credentials_output` above from what I can tell

Reply:
Yeah, these are a bit redundant here since the `existing_<item>_output` will still be populated even if there are no orgs/credentials. Cleaned these out.